convex/lp/simplex.go

```diff
@@ -349,7 +352,8 @@ func replaceBland(A mat64.Matrix, ab *mat64.Dense, xb []float64, basicIdxs, nonB
 			}
 			copy(biCopy, basicIdxs)
 			biCopy[replace] = nonBasicIdx[minIdx]
-			abTmp := extractColumns(A, biCopy)
+			abTmp := mat64.NewDense(m, len(biCopy), nil)
```

Comment:
I know this can be refactored out of the loop, but I'd rather do it as an optimization pass on replaceBland and keep this PR about extractColumns